### PR TITLE
Fix RR_1_1 to match spec and bring all-clusters app inline with minimums requirements

### LINKED ISSUE #25532: [BUG] TC_RR_1_1 group sub-test not run if Groups cluster endpoint is not 0
### Reproduction steps

- have Groups cluster configured on an endpoint different than 0
- run TC_RR_1_1.py test - will skip Groups cluster tests, due to no ACL access (ACL admin privileges are always on endpoint 0)

As it is not mandatory to have the Groups cluster on endpoint 0, this is an issue, as it will not test the groups and group key management functionality.

### Bug prevalence

every time

### GitHub hash of the SDK that was being used

b89e83be43dde7a79d90823f4bc1279eb53f76de

### Platform

python

### Platform Version(s)

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by tehampson**
Fixes: https://github.com/project-chip/connectedhomeip/issues/25532

Changes to RR_1.1:
This bring code in line to what is actually written in the RR_1.1 spec. There was a bug in add_all_groups that wrote all wrote all the groups to each of the endpoints instead, leading to writing more then the maximum of supported groups per fabric. This also incorporates test spec changes from https://github.com/CHIP-Specifications/chip-test-plans/pull/2503

Changes outside of RR_1.1:
* Changes required to bring all clusters up to compliance with minimum spec based on it having 3 Groups clusters
* Changes to yaml test run against all clusters app in CI now that there are more groups per fabric

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

File: examples/all-clusters-app/cc13x2x7_26x2x7/main/include/CHIPProjectConfig.h [hunk 1]
```diff
@@ -117,4 +117,8 @@
  */
 #define CHIP_IM_MAX_NUM_WRITE_HANDLER 1
```
Comment (tehampson) [inline examples/all-clusters-app/cc13x2x7_26x2x7/main/include/CHIPProjectConfig.h:119]: Add Blocking comment until https://github.com/CHIP-Specifications/chip-test-plans/pull/2503 is merged

File: src/python_testing/TC_RR_1_1.py [hunk 2]
```diff
@@ -442,31 +418,33 @@ async def test_TC_RR_1_1(self):
             await self.fill_user_label_list(dev_ctrl, self.dut_node_id)
         else:
             logging.info("Step 9: Skipped due to no UserLabel cluster instances")
-
-        # Step 10: Count all group cluster instances
+        # Step 10: Reconfig ACL to allow test runner access to Groups clusters on all endpoints.
+        logging.info("Step 10: Reconfiguring ACL to allow access to Groups Clusters")
+        self.send_acl(test_step=10, client_by_name=client_by_name, enable_access_to_group_cluster=False, fabric_table=fabric_table)
```
Comment (doublemis1) [inline src/python_testing/TC_RR_1_1.py:423]: enable_access_to_group_cluster shall be True?

File: src/python_testing/TC_RR_1_1.py [hunk 3]
```diff
@@ -747,9 +759,13 @@ def build_acl(self):
...
 
+        admin_target_field_2 = Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC00, deviceType=0xFFF1_BC30)
+        if enable_access_to_group_cluster:
+            admin_target_field_2 = Clusters.AccessControl.Structs.Target(cluster=0x0000_0004)
+
         admin_targets = [
             Clusters.AccessControl.Structs.Target(endpoint=0),
-            Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC00, deviceType=0xFFF1_BC30),
+            admin_target_field_2,
             Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC01, deviceType=0xFFF1_BC31)
```
Comment (doublemis1) [inline src/python_testing/TC_RR_1_1.py:769]: If we wanna be precise regarding Master test plan the Targets shall be  
`Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC00, deviceType=0xFFF1_FC31)`

File: src/python_testing/TC_RR_1_1.py [hunk 4]
```diff
@@ -747,9 +759,13 @@ def build_acl(self):
         # Administer ACL entry
         admin_subjects = [0xFFFF_FFFD_0001_0001, 0x2000_0000_0000_0001, 0x2000_0000_0000_0002, 0x2000_0000_0000_0003]
 
+        admin_target_field_2 = Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC00, deviceType=0xFFF1_BC30)
```
Comment (doublemis1) [inline src/python_testing/TC_RR_1_1.py:762]: Based on Master test plan RR-1.1 definition it should be Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC00, deviceType=0xFFF1_FC30)

File: src/python_testing/TC_RR_1_1.py (hunk 2 above)
Comment (tehampson) [reply src/python_testing/TC_RR_1_1.py:423]: Ops... that is correct

**Pull request merged**